### ISSUE TITLE
Val'anyr Hammer of the Ancient Kings Quest Item Bugged

### ISSUE THREAD

**Issue by lefty2108** [dalaranwow/dalaran-wow#5522]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Followed the quest line exactly; did Yogg 25 minus 1 keeper and attempted to throw the unbound frags into Yogg's mouth while he cast deafening roar. The item never went on cool down, Yogg never got the debuff, and the quest item that drops after killing Yogg never did.
**Expected behaviour**: Tell us what should happen instead
Yogg is supposed to drop a quest item that is returned to the console in the Iron Council room. This quest item is turned in and you're given Val'anyr
**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by lefty2108** [comment]
![wowscrnshot_061518_204247](https://user-images.githubusercontent.com/40413621/41633628-9fe138ee-73f4-11e8-94f0-d2a86c9038fd.jpg)
